content/en/integrations/guide/oracle-check-upgrade-7.50.1.md

```diff
@@ -21,6 +21,8 @@ The [Oracle integration][3] is completely rewritten in [Agent release][2] `7.53.
 
 The configuration in the subdirectory `oracle.d` remains the same.
 
+**Note**: The Oracle Real Application Cluster (RAC) customers must configure the Agent for each RAC node, because the Agent collects information from every node separately by querying `V$` views. The Agent doesn't query any `GV$` views to avoid generating interconnect traffic.
```

Comment:
```suggestion
**Note**: Oracle Real Application Cluster (RAC) customers must configure the Agent for each RAC node, because the Agent collects information from every node separately by querying `V$` views. The Agent doesn't query any `GV$` views to avoid generating interconnect traffic.
```